src/frontend/typer/sanity/tekstInnhold.ts

```diff
@@ -31,3 +32,11 @@ export interface IForsideTekstinnhold {
     utvidetBarnetrygdAlert: LocaleRecordBlock;
     soekerDuUtvidet: ISanitySpørsmålDokument;
 }
+
+export interface IFormateringsfeilmeldingerTekstinnhold {
+    datoErEtterSisteGyldigeTidspunkt: LocaleRecordString;
+    datoErForForsteGyldigeTidspunkt: LocaleRecordString;
+    ugyldigManed: LocaleRecordString;
+    datoKanIkkeVareIFortid: LocaleRecordString;
+    datoKanIkkeVareIFremtid: LocaleRecordString;
```

Comment:
Foretrekker om vi også endrer disse til å ikke ha "ikke" i seg 😅

Altså kanKunVelgeFortid og kanKunVelgeFremtid

Reply:
Det var det jeg også ville ha i starten, men logikken på disse er som regel at man kan velge a) dagens dato eller i fortid, eller b) dagens dato og i fremtid. Det blir i så fall veldig lange variabelnavn. Derfor gikk jeg for "ikke"-varianten 😅